### DISCUSSION TITLE
Building process of material design repository can not be implemeneted properly in windows

### DISCUSSION THREAD

**Discussion by amirtbi** [material-components/material-web#5543] [General]
I am trying to build the catalog folder of material=web [material design](https://github.com/material-components/material-web) in windows. When i run the command build in bash terminal of vsCodes, some imports such as ./materials-collection.ts path is not recognized. this problem is not existed in linux systems. How can solve it? Apparently, all scripts codes that include path, are not valid for windows!.

**Comment by asyncliz** [comment]
The project is not set up native Windows build support. You may need to use tools like git-bash or [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) to run the build commands.

**Reply by amirtbi** [reply]
Thanks,
